facturacion_mexico/facturacion_fiscal/tests/test_ffm_reconciliation.py

```
@@ -0,0 +1,530 @@
+			"docstatus": 1,
+		}
+	)
+	si.flags.ignore_validate = True
+	si.flags.ignore_mandatory = True
+	si.flags.ignore_links = True
+	si.db_insert()
+	frappe.db.commit()
+	return si.name
+
+
+def _seed_ffm(sales_invoice, status, *, facturapi_id="FA-1", uuid="U-1", sync="pending", last_sync=None):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_seed_ffm`

(ANN202)